cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/loadbalancer/AzureLoadBalancerProbe.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.cloudbreak.cloud.azure.loadbalancer;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.Objects;
+
+public class AzureLoadBalancerProbe {
+    private final int port;
+
+    private final String name;
+
+    public AzureLoadBalancerProbe(int port) {
+        this(port, "port-" + Integer.toString(port) + "-probe");
+    }
+
+    public AzureLoadBalancerProbe(int port, String name) {
+        Objects.requireNonNull(name);
+
+        this.port = port;
+        this.name = name;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Ah, sorry, I did a quick ctrl + f for `toString` methods and found e.g. `CloudInstance`, `CloudResourceStatus`, and a few others that use string builders.

I'll replace it with the IntelliJ generated one though, since hand rolling toString is dumb.

Thanks!